examples/python/invoke-rest-api/metaconfig-pduty.json

```diff
@@ -0,0 +1,60 @@
+{
+    "url": "https://events.pagerduty.com/v2/enqueue",
+    "headers": {
+        "content-type": "application/json; charset=UTF-8"
+    },
+    "auth": {},
```

Comment:
Is it assumed that users understand if and what to provide respective to authentication for each of these metaconfigs? I cannot find anything in the README.

Reply:
Yes, most apis provide some sort of api key to use and don't seem to require an auth. With some others, they might require basic authentication which can be send in the header. 

Where the apis require username and password explicitly, i've added that as an option in the config. I only provide limited support in terms of the possible auth options. I'll document this in the readme so there is no ambiguity.